pmg/views.py

```diff
@@ -407,19 +407,18 @@ def committee_detail_follow_cte(committee_id):
     return redirect(url_for('committee_detail', committee_id=committee_id))
 
 
-@app.route('/attendance-overview')
+@app.route('/archived-attendance-overview')
 @cache.memoize(
     make_name=lambda fname: cache_key(request),
     unless=lambda: should_skip_cache(request, current_user))
-def attendance_overview():
+def archived_attendance_overview():
     """
-    Display overview of attendance for meetings.
+    Show archived attendance view
     """
-    month = datetime.today().month
-    this_year = datetime.today().year
-    last_year = this_year - 1
+    this_year = 2019
```

Comment:
how are we ensuring we fix this? Is there a story that is very clearly something that can't be forgotte int he backlog?

Reply:
oh wait, this is archived - so for the next 5 years these years should work jsut fine?

Reply:
Correct, by archived only shows for the previous year, which doesn't really make sense anymore. It should be an archive for the entire history of the parliament. So we'll need to write something for that, I've added a Pivotal for that.